crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -717,7 +717,7 @@ pub struct RethModuleRegistry<Provider, Pool, Network, Tasks, Events> {
     /// Additional settings for handlers.
     config: RpcModuleConfig,
     /// Holds a clone of all the eth namespace handlers
-    eth: Option<EthHandlers<Provider, Pool, Network, Events>>,
+    pub eth: Option<EthHandlers<Provider, Pool, Network, Events>>,
```

Comment:
we don't want to expose internals here
```suggestion
   eth: Option<EthHandlers<Provider, Pool, Network, Events>>,
```